kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+
+  <KPageContainer>
+
+    <h1>{{ $tr('syncData') }}</h1>
+    <span>
+      {{ $tr('access') }}
+      <KButton
+        appearance="basic-link"
+        :text="$tr('learnMore')"
+        @click="displayModal(Modals.PRIVACY)"
+      />
+    </span>
+
+    <CoreTable>
+      <thead slot="thead">
+        <tr>
+          <th>{{ $tr('facility') }}</th>
+        </tr>
+      </thead>
+      <transition-group slot="tbody" tag="tbody" name="list">
+        <tr v-for="facility in facilities" :key="facility.id">
+          <td>
+            <span>
+              <h2 class="name">{{ facility.name }}</h2>
+              <UiIcon v-if="facility.dataset.registered" ref="icon">
+                <mat-svg
+                  name="verified_user"
+                  category="action"
+                  :style="{fill: $themePalette.green.v_500}"
+                />
+              </UiIcon>
+              <KTooltip
+                reference="icon"
+                :refs="$refs"
+              >
+                {{ $tr('registeredAlready') }}
+              </KTooltip>
+            </span>
+            <span class="synced">
+              <span>
+                <template v-if="facility.syncing">
+                  <KCircularLoader class="loader" :size="16" :delay="false" />
+                  {{ $tr('syncing') }}
+                </template>
+                <template v-else>
+                  <template v-if="facility.last_sync_failed">
+                    {{ $tr('syncFailed') }}
+                  </template>
+                  <template v-if="facility.last_synced === null">
+                    {{ $tr('neverSynced') }}
+                  </template>
+                  <template v-else>
+                    {{ $tr('lastSync') }} {{ formattedTime(facility.last_synced) }}
+                  </template>
+                </template>
+              </span>
+            </span>
+          </td>
+          <td>
+            <KButton
+              class="register"
+              appearance="raised-button"
+              :text="$tr('register')"
+              :disabled="facilityTaskId !== ''"
+              @click="register(facility)"
+            />
+            <KButton
+              class="sync"
+              appearance="raised-button"
+              :text="$tr('sync')"
+              :disabled="facilityTaskId !== ''"
+              @click="sync(facility)"
+            />
```

Comment:
Sync should be disabled until it's been successfully registered

![image](https://user-images.githubusercontent.com/2367265/68334053-add51380-008e-11ea-8be4-dc24e401373d.png)

Reply:
hmm, is this actually the intended use case? cc: @jamalex 
because you can sync without registering. So unsure if we want to limit sync's to only our partners to whom we give out project tokens, or if we want to allow anyone to initially sync their data to us.

Reply:
But then again, it could be useful to have a "private" release to our partners, so we can uncover any issues before we allow the masses to start syncing to us.

Reply:
> unsure if we want to limit sync's to only our partners to whom we give out project tokens, or if we want to allow anyone to initially sync their data to us.

From the requirements in the [project](https://www.notion.so/learningequality/Kolibri-data-syncing-0-13-0-MVP-f688948206eb45359d653552519e9f79):

> Invitation-only. User must use a project token - facilities cannot be synced anonymously